rita_common/src/usage_tracker/mod.rs

```diff
@@ -427,8 +455,6 @@ pub fn save_usage_on_shutdown() {
     let history = &mut USAGE_TRACKER.write().unwrap();
     history.last_save_hour = current_hour;
     trim_payments(MAX_TX_ENTRIES, &mut history.payments);
-    let res = history.save();
-    info!("Shutdown: saving usage data: {:?}", res);
```

Comment:
shutdown save for usage tracker and debt keeper should be kept. Routers in the field rarely get explicit restart instructions. I guess this is is the one exception to the push for saving all in one place, we can check if the router has small storage and disable it then.